src/core/transposition.h

```diff
@@ -15,9 +15,40 @@ enum TtFlag : uint8_t {
     TtBeta = 2,
 };
 
+/* Storage for smaller (< 8 bits) types stored in TT data
+ * 0b00000011 -> flag (2 bits)
+ * 0b00000100 -> ttPv (1 bit) */
+struct TtInfo {
+    TtInfo() = default;
+    TtInfo(TtFlag flag, bool isPv)
+        : data((flag & s_flagMask)
+              | (isPv << s_ttPvShift))
+    {
+    }
+
+    /* tt flag - which bound the entry has */
+    inline TtFlag flag() const
+    {
+        return static_cast<TtFlag>(data & s_flagMask);
+    }
+
+    /* flag to indicate if this entry is following the PV line */
+    inline bool ttPv() const
```

Comment:
Do you think there's maybe a better name for this? Perhaps `isPv` or similar?

Reply:
I was a little afraid of calling it that as we already have a "isPv" flag which I don't want it to be mixed up with.
I can simply call it `pv` here (where it's bound to TT) and use `ttPv` in the searcher to make it explicit there? :)